test/configCases/css/css-import/style.css

```diff
@@ -0,0 +1,192 @@
+@import "print.css?foo=1";
+@import url("print.css?foo=2");
+@import "print.css?foo=3" layer(default);
+@import url("print.css?foo=4") layer(default);
+@import "print.css?foo=5" supports(display: flex);
+@import url("print.css?foo=6") supports(display: flex);
+@import "print.css?foo=7" screen and (min-width: 400px);
+@import url("print.css?foo=8") screen and (min-width: 400px);
+@import "print.css?foo=9" layer(default) supports(display: flex);
+@import "print.css?foo=10" layer(default) screen and (min-width: 400px);
+@import "print.css?foo=11" supports(display: flex) screen and (min-width: 400px);
+@import "print.css?foo=12" layer(default) supports(display: flex) screen and (min-width: 400px);
+@import "print.css?foo=13"layer(default)supports(display: flex)screen and (min-width: 400px);
+@import url(print.css?foo=14)layer(default)supports(display: flex)screen and (min-width: 400px);
+@import url("print.css?foo=15")layer(default)supports(display: flex)screen and (min-width: 400px);
+@import url(print.css?foo=16)layer(default)supports(background: url(./img.png))screen and (min-width: 400px);
+@import url(print.css?foo=17)layer(default)supports(background: url("./img.png"))screen and (min-width: 400px);
+@import url(print.css?foo=18)screen;
+@import url("print.css?foo=19")screen;
+@import "print.css?foo=20"screen;
+@import url(print.css?foo=18)   screen   ;
+@import url("print.css?foo=19")   screen   ;
+@import "print.css?foo=20"   screen  ;
+@import "print.css?foo=21"    ;
+
+/* Has the same URL */
+@import "imported.css";
+@import "imported.css" layer(base);
+@import "imported.css" supports(display: flex);
+@import "imported.css" screen, print;
+
+@import url(style2.css?foo=1);
+@import url('style2.css?foo=2');
+@import url("style2.css?foo=3");
+@IMPORT url(style2.css?foo=4);
+@import URL(style2.css?foo=5);
+@import url(style2.css?foo=6 );
+@import url( style2.css?foo=7);
+@import url( style2.css?foo=8 );
+@import url(
+style2.css?foo=9
+);
+/*@import url();
+@import url('');
+@import url("");
```

Comment:
Are these future style cases we need to enable?

Reply:
Yeah, I am working on it, we should have zero commented `@imports` here